src/app/zap-templates/templates/app/MatterIDL.zapt

```diff
@@ -63,14 +65,14 @@
       {{/unless}}
     {{/enabled_attributes_for_cluster_and_side~}}
 
+  {{~/if}}
```

Comment:
Why did this move?  client-side clusters do not have eventList and whatnot.

Reply:
This is because ZAP marks all global attributes optional. I did not want idl to say that ... since all except of eventList are actually mandatory and reported (event list should be too ... but no cluster currently implements it ... so it is effectively optional)

Reply:
Client cluster here means "clients can ask for this from the server". Even previous .matter files were showing them, but because optional was not taken into about all was well.

Once I considered optional, if I keep the old way, client clusters have all global attributes flagged optional. This may be a zap or xml issue, did not check it.

Reply:
> This is because ZAP marks all global attributes optional

Uh...  That seems like a bug.  They're all mandatory in Matter.

> so it is effectively optional

"optional" means you can choose whether to implement it or not.  That's not the story with EventList.  It's either implemented everywhere or implemented nowhere.  It's a mandatory feature that is not yet implemented.

> Client cluster here means "clients can ask for this from the server"

Ah, I see.  OK.